devlib/utils/android.py

```diff
@@ -216,7 +219,8 @@ def pull(self, source, dest, timeout=None):
         command = "pull '{}' '{}'".format(source, dest)
         return adb_command(self.device, command, timeout=timeout)
 
-    def execute(self, command, timeout=None, check_exit_code=False, as_root=False):
+    def execute(self, command, timeout=None, check_exit_code=False,
+                as_root=False, wrap_busybox=False, strip_colors=True):
```

Comment:
Removed these, it was a muddled up combination of workload automation code and my understanding of it.